{{cookiecutter.repo_name}}/Makefile

```diff
@@ -67,7 +67,7 @@ compile_translations:
 fake_translations: extract_translations dummy_translations compile_translations
 
 pull_translations:
-	tx pull -a
```

Comment:
Is there any reason why we shouldn't use the force flag (f) here, as we do in https://github.com/edx/cookiecutter-django-app/pull/18/files#diff-af4c0c482ca0bcd7910328df7120974dL88 ?